app/main.dev.js

```diff
@@ -361,7 +349,28 @@ app.on('ready', async () => {
       startLnd(options.alias, options.autopilot)
     } else {
       mainLog.info('CONNECTING TO CUSTOM LND INSTANCE')
-      startGrpc()
+      try {
+        await startGrpc()
```

Comment:
nit: Given you're catching the errors, it would be cleaner to use `.then`/`.catch` rather than immediately catch a `throw`.

Reply:
I think thats pretty subjective. In this codebase we are currently very inconsistent with regards to wether we use callbacks, promises, or async. IMO async generally allows for a cleaner and more universally standard syntax. But again, I think that is a little subjective.

I don't have too string an opinion either way so if the consensus is to stick with promises I can update it. Feels like a bit of a shame not to leverage nodes latest and greatest features though!